core/src/main/java/io/grpc/HandlerRegistry.java

```diff
@@ -42,6 +43,12 @@
 public abstract class HandlerRegistry {
 
   /**
+   * Returns the {@link ServerServiceDefinition}s provided by the registry.
+   */
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/2222")
+  public abstract Collection<ServerServiceDefinition> getServices();
```

Comment:
Any chance this could be a List or a Set?  Something that more strongly conveys what the type a user will get back?

It makes it easier to test (since you dont have to worry about order).